components/Quote/Quote.tsx

```diff
@@ -76,24 +77,24 @@ export const Quote = ({
             >
               {teaser}
             </Text>
-          </FlexBox>
-        )}
-        {body && (
-          <Text
-            weight="semibold"
-            variant={isLargeBody ? undefined : 'big'}
-            leading="display"
-            size={isLargeBody ? 2 : undefined}
-            font="serif"
-            className={styles.body}
-          >
-            {content}
-          </Text>
-        )}
-        {source && (
-          <Text variant="card" className={styles.source}>{source}</Text>
-        )}
-      </blockquote>
-    </Container>
-  </AnimateInView>);
+          )}
+          {body && (
+            <Text
+              weight="semibold"
+              variant={isLargeBody ? undefined : 'big'}
+              leading="display"
+              size={isLargeBody ? 2 : undefined}
+              font="serif"
+              className={styles.body(hasRealTeaser)}
+            >
+              {body}
```

Comment:
Is body not a rich text field anymore? How'd you handle the transition?

Reply:
This one was never a rich text field. I think during the POC stage, it was undecided, and we never removed the unnecessary support for rich text later on.